internal/workflow/preprocessing.go

```diff
@@ -67,7 +67,7 @@ func (w *PreprocessingWorkflow) Execute(ctx temporalsdk_workflow.Context, params
 
 	// TODO Make the file path a part of the enduro config or check the configuration later
 	// A succumb file works like a .gitignore file.
-	succumbPath := "./.succumb"
+	succumbPath := "/home/preprocessing-moma/.config/.succumb"
```

Comment:
I think this path should be configurable so it works in different deploy environments.

Reply:
sure at some point that will be helpful but this is proof of concept right now.